### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by qmarch** [MicrosoftDocs/windowsserverdocs#1766]
I have windows 10 home on my desktop and laptop but I would like to upgrade to pro and start using remote desktop feature. I want to be able to use the laptop to connect to the desktop when I am on the go. Will I need to upgrade to windows 10 pro on both machines or will I only need to upgrade one? If I only need to upgrade one, is it the computer I'm connecting to or the computer I'm connecting with that needs the pro license? Thanks. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 71cfbf1a-c5b7-df0f-8194-6ebc03afcc7d
* Version Independent ID: f36b567f-aa48-2c8f-4908-b1fe18cd2842
* Content: [Remote Desktop - Allow access to your PC](https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/remote-desktop-allow-access#feedback)
* Content Source: [WindowsServerDocs/remote/remote-desktop-services/clients/remote-desktop-allow-access.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/remote/remote-desktop-services/clients/remote-desktop-allow-access.md)
* Product: **windows-server-threshold**
* GitHub Login: @lizap
* Microsoft Alias: **elizapo**

**Comment by illfated** [comment]
You can use the Windows 10 Home client computer to run the Remote Desktop connection application to connect to a Windows 10 Pro, Enterprise or Windows Server device, without making any other changes than to check if you need to install the Remote Desktop connection application on Windows 10 Home.

If you search the web for terms including "Windows 10 Home" and "Remote Desktop connection" and variations thereof, you will most likely find pages on how to add the Remote Desktop services on Windows 10 Home as well, but that is not supported by Microsoft. (You can, of course, choose to experiment with RDP wrapper solutions yourself, if you are inclined to do a lot of testing.)

Basically, if you don't want to experiment with hackfix solutions, you need the RDP server services to run on the target computer, meaning you will need Windows 10 Pro or better on the target computer. You will also need to enable access (allow RDP connections) on the target computer, as well as making sure there is at least 1 user account allowed to log in on the target computer via the RDP connection. I presume you already know the details regarding network connection, TCP port(s) and firewall configurations needed to make that connection pass through to the target computer.

**Comment by illfated** [comment]
(Please close this ticket as answered and resolved. It has been open for so long, it is rather unlikely to receive a follow-up.)

**Comment by illfated** [comment]
#sign-off

**Comment by Heidilohr** [comment]
#please-close

**Comment by illfated** [comment]
#please-close
